### PR TITLE
feat: Expose Joi along with Bagger instance. Fixes #60

### LINKED ISSUE #60: Expose Joi along with Bagger instance
Bagger should expose the Joi which it's using to make it easier to use the compatible version. 
This means that any bagger user can just import it's own Joi instead of including yet another dependency.

### PR DESCRIPTION
**Pull request by Saeger**
Making Bagger exposes JOI by default. #60

### CODE REVIEW

**Comment by Saeger** [comment]
@juliuscc please review

File: README.md [hunk 1]
```diff
@@ -10,29 +10,26 @@ A joi-compatible tool for building Swagger (Open API 3) documents. It enables de
 ## Features
 
 - 🔨 **Builder pattern:** Dead simple api to create complex Swagger documents.
-- ✨ **joi compatibility:** Enables developers to use the same schemas for validation and documentation.
+- ✨ **joi compatibility:** Enables developers to use the same schemas for validation and documentation. Embeded Joi is now exposed by default.
```
Comment (tgun89) [inline README.md:13]:
```suggestion
- ✨ **joi compatibility:** Enables developers to use the same schemas for validation and documentation. Embedded Joi is now exposed by default.
```

File: README.md [hunk 2]
```diff
@@ -10,29 +10,26 @@ A joi-compatible tool for building Swagger (Open API 3) documents. It enables de
...
 - 🔎 **Intellisense:** Really nice intellisense suggestions, and TypeScript definitions.
 - 🔒 **Type safety:** Bagger always produces 100% valid Swagger documents. If you use TypeScript the compiler will enforce correctness in most cases, and otherwise Bagger will validate during compilation.
 
 ## Usage
 
 ```js
-// Use the default Bagger instance
-const bagger = require('@digitalroute/bagger').default;
+// Import as a classic javascript
+const bagger = require('@digitalroute/bagger').bagger;
```
Comment (tgun89) [inline README.md:21]:
```suggestion
const { bagger } = require('@digitalroute/bagger');
```

File: lib/bagger.ts [hunk 3]
```diff
@@ -192,5 +192,4 @@ export class Bagger {
   }
 }
 
-const defaultInstance = new Bagger();
-export default defaultInstance;
+export const defaultInstance = new Bagger();
```
Comment (tgun89) [inline lib/bagger.ts:195]:
```suggestion
export const bagger = new Bagger();
```

File: lib/index.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import { defaultInstance as bagger } from './bagger';
```
Comment (tgun89) [inline lib/index.ts:1]:
```suggestion
import { bagger } from './bagger';
```

File: rollup.config.ts [hunk 5]
```diff
@@ -7,7 +7,7 @@ import json from 'rollup-plugin-json';
 
 const pkg = require('./package.json');
 
-const libraryName = 'bagger';
+const libraryName = 'index';
```
Comment (tgun89) [inline rollup.config.ts:10]:
```suggestion
const libEntrypoint = 'index';
```

File: rollup.config.ts [hunk 6]
```diff
@@ -7,7 +7,7 @@ import json from 'rollup-plugin-json';
...
 
 export default {
   input: `lib/${libraryName}.ts`,
```
Comment (tgun89) [inline rollup.config.ts:13]:
```suggestion
  input: `lib/${libEntrypoint}.ts`,
```

File: test/configuration.test.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import bagger from '../lib/bagger';
+import { bagger } from '../lib';
 import joi from 'joi';
```
Comment (tgun89) [inline test/configuration.test.ts:2]: import joi from `lib` instead

**Review by tgun89 — CHANGES_REQUESTED** [review]

**Review by tgun89 — APPROVED** [review]
🚀

**Pull request merged**